### PR TITLE
docs(PPDSC-1976): Media list fix

### PR DESCRIPTION
**Pull request by Srinivasan-Ramasamy**
PPDSC-1976

**What**

1. MediaList component media should have borderRadius defaults all the corners across the application.

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on: 
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-1976-medialist-fix 
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-1976-medialist-fix/storybook

**Review by tbradbury — APPROVED** [review]

**Comment by Srinivasan-Ramasamy** [comment]
Thanks @tbradbury for reviewing the code :+1:

File: src/card/styled.tsx [hunk 1]
```diff
@@ -43,6 +44,9 @@ export const StyledCardContainerMedia = styled.div<
   box-sizing: border-box;
   display: block;
   position: relative;
+  svg {
+    ${getBorderCssFromTheme('borderRadius', 'borderRadiusRounded010')};
+  }
```
[the remark this answers is not in the document]
Comment (Vanals) [reply src/card/styled.tsx:49]: 🤔  I guess we do not support `svg` inside style presets atm? 
also not sure if we could add it as nested object in the base? @pdimova

[the remark this answers is not in the document]
Comment (Srinivasan-Ramasamy) [reply src/card/styled.tsx:51]: @mstuartf and @Vanals Thank you for your time for giving your valuable feedback.

This style changes has to be updated nested SVG. I was also initialy thinking the same, but after reffered some of the component styles files I got doubt is cleared and updated the same way which those component follows. For exanoke Toast and Flag component has the same nested structure in their styles files. 

Please suggest me if we need to do any changes?

[the remark this answers is not in the document]
Comment (mstuartf) [reply src/card/styled.tsx:51]: 👍 If this is a pattern that we follow elsewhere then it's fine for me. But I would be interested to understand why it doesn't work using defaults / style presets.

[the remark this answers is not in the document]
Comment (Srinivasan-Ramasamy) [reply src/card/styled.tsx:51]: Ok @mstuartf, thanks.

**Review by mstuartf — APPROVED** [review]

**Review by Xin00163 — APPROVED** [review]

**Pull request merged**